weather_mv/loader_pipeline/bq_test.py

```diff
@@ -222,9 +232,36 @@ def test_extract_rows_nan(self):
             'time': '2018-01-02T06:00:00+00:00',
             'u10': None,
             'v10': 0.03294110298156738,
+            'geo_point': geojson.dumps(geojson.Point((-108.0, 49.0))),
         }
         self.assertRowsEqual(actual, expected)
 
+    def test_extract_rows__with_valid_lat_long(self):
+        valid_lat_long = [[-90, -360], [-90, -359], [-45, -180], [-45, -45], [0, 0], [45, 45], [45, 180], [90, 359],
+                          [90, 360]]
+        actual_val = [
+            '{"type": "Point", "coordinates": [0, -90]}',
+            '{"type": "Point", "coordinates": [1, -90]}',
+            '{"type": "Point", "coordinates": [-180, -45]}',
+            '{"type": "Point", "coordinates": [-45, -45]}',
+            '{"type": "Point", "coordinates": [0, 0]}',
+            '{"type": "Point", "coordinates": [45, 45]}',
+            '{"type": "Point", "coordinates": [-180, 45]}',
+            '{"type": "Point", "coordinates": [-1, 90]}',
+            '{"type": "Point", "coordinates": [0, 90]}'
+        ]
+        for actual, (lat, long) in zip(actual_val, valid_lat_long):
+            with self.subTest():
+                expected = fetch_geo_point(lat, long)
+                self.assertEqual(actual, expected)
+
+    def test_extract_rows__with_invalid_lat(self):
+        invalid_lat_long = [[-100, -2000], [-100, -500], [100, 500], [100, 2000]]
+        for i, (lat, long) in enumerate(invalid_lat_long):
```

Comment:
Nit: It looks like the `i` variable is unused here.

Reply:
Suggested code-changes completed.